app/build.gradle

```diff
@@ -50,8 +50,28 @@ dependencies {
     def recyclerview_version = "1.1.0"
     implementation "androidx.recyclerview:recyclerview:$recyclerview_version"
 
-    //glide
+    //Glide
     def glide_version = "4.11.0"
     implementation "com.github.bumptech.glide:glide:$glide_version"
     annotationProcessor "com.github.bumptech.glide:compiler:$glide_version"
+
+    //Retrofit
+    def retrofit_version = "2.0.0"
+    implementation "com.squareup.retrofit2:retrofit:$retrofit_version"
+
+    //Retrofit converter Gson
+    def retrofit_converter_gson = "2.0.0"
+    implementation "com.squareup.retrofit2:converter-gson:$retrofit_converter_gson"
+
+    // RxAndroid
+    def rxandroid_version = '2.0.1'
+    implementation "io.reactivex.rxjava2:rxandroid:$rxandroid_version"
+
+    // RxJava
+    def rxjava_version = '2.1.7'
+    implementation "io.reactivex.rxjava2:rxjava:$rxjava_version"
+
+    // Rx-Retrofit Call Adapter
+    def rxcalladapter_version = '2.5.0'
```

Comment:
This is not the right way to put the number version of the dependencies. You have two alternatives, the first is move all the number version into the EXT{ ... } in the main project. The second, is to create a dependencies.gradle to have all the dependencies. If you need help, call me .

Reply:
Done, added the second way of doing it